server/models/visits.js

```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var q = require('q');
+
+var config = require('../config');
+var postgres = require('../db/postgres');
+var elasticsearch = require('../db/elasticsearch');
+var log = require('../logger')('server.models.visits');
+var visit = require('./visit');
+
+var _verbose = function() {
+  var logline = [].join.call(arguments, ', ');
+  log.verbose(logline);
+};
+
+var visits = {
+  _onFulfilled: function _onFulfilled(msg, callback, results) {
+    _verbose(msg);
+    callback(null, results);
+  },
+  _onRejected: function _onRejected(msg, callback, err) {
+    log.warn(msg);
+    callback(err);
+  },
+  _transform: function _transform(results) {
+    if (!results || !results.length) { return; }
+    var transformed = [];
+    results.forEach(function(result) {
+      transformed.push(visit._transform(result));
+    });
+    return transformed;
```

Comment:
Same silly comment as https://github.com/mozilla/chronicle/pull/224/files#r23810436
I think ESLint is warning against returning an Object on line 35 vs a naked `return` on line 30 (which would translate to `return undefined`).